mne/viz/_3d.py

```diff
@@ -481,11 +482,25 @@ def plot_alignment(info, trans=None, subject=None, subjects_dir=None,
         `'$SUBJECTS_DIR/$SUBJECT/bem/$SUBJECT-$SOURCE.fif'`, and then look for
         `'$SUBJECT*$SOURCE.fif'` in the same directory. For `'outer_skin'`,
         the subjects bem and bem/flash folders are searched. Defaults to None.
+    show_axes : bool
+        If True (default False), coordinate frame axis indicators will be
+        shown:
+
+        * head in pink
+        * MRI in gray (if ``trans is not None``)
+        * MEG in blue (if MEG sensors are present)
+
+        .. versionadded:: 0.16
     fig : mayavi figure object | None
         Mayavi Scene (instance of mlab.Figure) in which to plot the alignment.
         If ``None``, creates a new 600x600 pixel figure with black background.
 
         .. versionadded:: 0.16
+    interaction : str
+        Can be "trackball" (default) or "terrain", i.e. a turntable-style
+        camera.
+
+        .. versionadded:: 0.16
```

Comment:
If there's only two options, might also be worth having a boolean choice (`trackball_interaction=bool`)

Reply:
This mirrors the PySurfer interface, IIRC we wanted the `str` to leave open future possibilities (while still being as readable/just a touch less usable)